bin/scripts/config-helper.ts

```diff
@@ -0,0 +1,157 @@
+const fs = require('fs');
+const figlet = require("figlet");
+const { prompt } = require('enquirer');
+
+
+// These properties must always be set
+let questions = [
```

Comment:
I was thinking of doing that, but you can see further down that in the case the user selects `Advanced` we just append the advanced questions to this list of questions since we *always* have to ask the basic questions.

So naming this list to something like `basic_questions` seemed confusing when you get to this line:

```
const answers = await prompt(questions);
```

It would be misleading to make it:

```
const answers = await prompt(basic_questions);
```

Since it could contain basic *and* advanced questions.